components/inputs/input-time.js

```diff
@@ -92,13 +123,40 @@ class InputTime extends LitElement {
 
 	render() {
 		const input = html`
-			<input
-				aria-label="${ifDefined(this._getAriaLabel())}"
-				@change="${this._handleChange}"
-				class="d2l-input"
-				?disabled="${this.disabled}"
-				@keypress="${this._handleKeypress}"
-				.value="${this._formattedValue}">
+			<d2l-dropdown>
+				<div
+					id="combo-container"
+					role="combobox"
+					aria-owns="time-menu"
+					class="d2l-dropdown-opener"
+					aria-expanded="false">
+					<input
+						aria-controls="time-menu"
+						aria-label="${ifDefined(this._getAriaLabel())}"
+						@change="${this._handleChange}"
+						class="d2l-input"
+						?disabled="${this.disabled}"
+						@keypress="${this._handleKeypress}"
+						.value="${this._formattedValue}">
+				</div>
+				<d2l-dropdown-menu id="dropdown" no-padding-footer min-width="195">
+					<d2l-menu
+						id="time-menu"
+						role="listbox"
+						class="d2l-input-time-menu"
+						aria-label="${ifDefined(this.label)}"
```

Comment:
Would using `aria-labelledby` and pointing at the `<span class="d2l-input-label">` (you'll need to put an ID on it) work as well? Reduces the number of times the label text is repeated.

Reply:
I think the concern is that if we're using `label-hidden` that span doesn't exist 🤷‍♀

Reply:
Gotcha yep, sorry forgot that was conditionally rendered.